modules/renter/download.go

```diff
@@ -493,11 +518,26 @@ func (r *Renter) managedScheduleNewChunks(ds *downloadState) {
 			continue
 		}
 
+		// Determine whether the download should be sped up.
+		su := (nextChunk.download.numChunks - nextChunk.index) <= uint64(numFastChunks)
+		var np = nextChunk.download.erasureCode.MinPieces()
```

Comment:
variable names need to read like english. There are exceptions for 'obvious context', but I don't think that applies here.

For example, `cst := newConsensusSetTester` - there's an obvious mapping from cst to ConsensusSetTester. But above, `su` doesn't obviously map to anything.

Same with `np`, it's a bit closer because you have the context of 'MinPieces', but I think still difficult to read.

Also, convention would be to do `np := nextChunk...`, instead of declaring the variable with `var`.